docs/migrating_to_9.md

```diff
@@ -239,6 +239,26 @@ await test.save();
 test.uuid; // string
 ```
 
+### SchemaType caster and casterConstructor properties were removed
+
+In Mongoose 8, certain schema type instances had a `caster` property which contained either the embedded schema type or embedded subdocument constructor.
+In Mongoose 9, to make types and internal logic more consistent, we removed the `caster` property in favor of `embeddedSchemaType` and `Constructor`.
```

Comment:
Maybe this should also mention the rename from `$embeddedSchemaType`?

Reply:
Fixed in https://github.com/Automattic/mongoose/pull/15513/commits/c5108117fabfa961872bcd988939274b25a5270a